common/src/test/scala/org/typelevel/discipline/munit/Discipline.scala

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package org.typelevel.discipline
+package munit
+
+import _root_.munit.Location
+import _root_.munit.ScalaCheckSuite
+
+trait Discipline extends ScalaCheckSuite {
```

Comment:
We may want to move this to tyelevel in the future

Reply:
Ok, I'm not sure if I'm missing something here but it works for now. I'll start by moving it to a separate project.